src/main/java/org/spongepowered/common/event/tracking/phase/packet/PacketState.java

```diff
@@ -94,7 +95,9 @@ public void addNotifierToBlockEvent(P context, IMixinWorldServer mixinWorldServe
     public void associateNeighborStateNotifier(P unwindingContext, BlockPos sourcePos, Block block, BlockPos notifyPos, WorldServer minecraftWorld,
         PlayerTracker.Type notifier) {
         final Player player = unwindingContext.getSpongePlayer();
-        ((IMixinChunk) minecraftWorld.getChunkFromBlockCoords(notifyPos)).setBlockNotifier(notifyPos, player.getUniqueId());
+        Chunk chunk = minecraftWorld.getChunkFromBlockCoords(notifyPos);
+        ((IMixinChunk) chunk).setBlockCreator(notifyPos, player.getUniqueId());
```

Comment:
Why are we setting the block creator here? Just found where this came from and I'm not following why we're notifying blocks makes the player the now creator, when they haven't created the blocks in the first place (unless a block change is done on that block).

Reply:
From my understanding, what this is now doing is making any blocks notified by a player interacting with the block as *created* by the player, when that is simply not the case nor intention of such in the API.

Reply:
Ver much a bug. Will need to eliminate this line here and re-test for this PR's original intention of fixing blocks falling without an owner.
![2018-03-20_22 41 11](https://user-images.githubusercontent.com/1203877/37695896-f827e450-2c8f-11e8-965a-a6f5be3a2f32.png)
![2018-03-20_22 41 17](https://user-images.githubusercontent.com/1203877/37695897-f842371a-2c8f-11e8-9552-9fa92f91e322.png)
![2018-03-20_22 41 22](https://user-images.githubusercontent.com/1203877/37695898-f85b3cc4-2c8f-11e8-9c0b-54d7c87dd74a.png)

Reply:
Working on it. For now, this is what I came up with https://github.com/ImMorpheus/SpongeCommon/commit/d807846c30fc5e3a49fc8590d5f3e49dcc29f600